package.json

```diff
@@ -142,7 +142,7 @@
     "react-native-screen-brightness": "^2.0.0-alpha",
     "react-native-screens": "^2.18.1",
     "react-native-sha256": "1.2.3",
-    "react-native-share": "^3.3.2",
+    "react-native-share": "^7.3.5",
```

Comment:
Since from version `6.x` the project migrated to TypeScript I think this could be a great occasion to remove `"@types/react-native-share": "^3.1.0"` that are now outdated. What you think?

Reply:
Great point! 09624b310dcf05af0f31fc3d010545ed633a50a2